tests/ert_tests/data/test_loader.py

```diff
@@ -22,7 +22,7 @@ def mocked_obs_node_get_index_nr(nr):
     ["GEN_OBS", "SUMMARY_OBS"],
 )
 def test_data_loader_factory(obs_type):
-    # pylint: disable=comparison-with-callable
+    # pylint: disable=comparison-with-callable, no-member
```

Comment:
Same comment as above: What do you think about adding a comment on why no-member is being disabled here?